### ISSUE TITLE
Docker image version 0.13.0-beta2 just got tagged as terraform:light

### ISSUE THREAD

**Issue by rdumont** [hashicorp/terraform#25279]
The way I see it, `terraform:light` was supposed to be a stable tag, and until 30 minutes ago it contained version 0.12.x.

Below is a screenshot from Docker Hub that shows the same digest for both `light` and `0.13.0-beta2`.

![image](https://user-images.githubusercontent.com/430448/84954586-057b3900-b0cc-11ea-92f6-3e405ab5f41f.png)

### Expected Behavior

I expected `terraform:light` to contain a stable 0.12 release.

### Actual Behavior

The image tagged `terraform:light` contains a pre-release 0.13.0-beta2.

**Comment by danieldreier** [comment]
this was not intentional - thank you for reporting. We're on it.

**Comment by danieldreier** [comment]
We have fixed this and updated our release process to avoid this in the future. `terraform:light` now matches `terraform:latest` again. I understand that people use these versions and expect them to be the latest GA version, so I take this seriously and I apologize for any problems this has caused.

The root cause here is that we had previously done pre-releases by hand, and are now automating them as part of our release CI process. In automating it, we made a mistake, and accidentally pushed this pre-release. Now that this is automated, I feel confident that we won't make a similar mistake in the future.

If you do use this, be aware that in about a month when 0.13.0 GA is released, `terraform:light` will be updated to 0.13.0.
